File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/network/ResponseModels/AutoCompleteResponse.java
package com.mowdowndevelopments.blurb.network.ResponseModels;

import com.squareup.moshi.Json;

public class AutoCompleteResponse {
    @Json(name = "label")
    private String feedTitle;
    @Json(name = "num_subscribers")
    private int subscriberCount;
    private String tagline;
    @Json(name = "value")
    private String url;

    public AutoCompleteResponse(String feedTitle, int subscriberCount, String tagline, String url) {
        this.feedTitle = feedTitle;
        this.subscriberCount = subscriberCount;
        this.tagline = tagline;
        this.url = url;
    }

    public int getSubscriberCount() {
        return subscriberCount;
    }

    public String getFeedTitle() {
        return feedTitle;
    }

    public String getTagline() {
        return tagline;
    }

    public String getUrl() {
        return url;
    }

}

File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/network/responseModels/AutoCompleteResponse.kt
package com.mowdowndevelopments.blurb.network.responseModels

import com.squareup.moshi.Json

data class AutoCompleteResponse(@field:Json(name = "label") val feedTitle: String,
                                @field:Json(name = "num_subscribers") val subscriberCount: Int,
                                val tagline: String,
                                @field:Json(name = "value") val url: String)